libp2p/Host.cpp

```diff
@@ -919,76 +910,50 @@ void Host::restoreNetwork(bytesConstRef _b)
     if (!isStarted())
         BOOST_THROW_EXCEPTION(NetworkStartRequired());
 
-    if (m_dropPeers)
-        return;
-    
     RecursiveGuard l(x_sessions);
     RLP r(_b);
-    unsigned fileVersion = r[0].toInt<unsigned>();
-    if (r.itemCount() > 0 && r[0].isInt() && fileVersion >= dev::p2p::c_protocolVersion - 1)
+    unsigned protocolVersion = r[0].toInt<unsigned>();
```

Comment:
clang-tidy suggest not to repeat the type if obvious from the expression. Use `auto`?